### ISSUE TITLE
拆分训练集和测试集

### ISSUE THREAD

**Issue by zhaobenshan11** [lihaolin88/3D-Half-Gaussian-Splatting#7]
Hello 
     The dataset that has just been put down, for example, mip-nerf360, can't you use it directly, python train.py -s data/360_v2/counter -m output/counter, isn't this okay, if not, then how to split the training set and the test set.

**Comment by lihaolin88** [comment]
Hello, thank you for your interest! Yes, you can use the code run with the original dataset with the command "python train.py -s data/360_v2/counter --eval", just run with a "--eval" setting, and then the code will automatically split train and test.
